mixins/scrollBetweenSections.ts

```diff
@@ -16,12 +16,12 @@ export default class ScrollBetweenSections extends Vue {
         rootMargin: windowTriggerMargins,
         threshold
       }
-    );
-    (this.$el as HTMLElement)
-      .querySelectorAll('.scrollable')
-      .forEach((section) => {
-        (this._observer as IntersectionObserver).observe(section)
-      })
+    )
+    this.updateObserved()
+  }
+
+  updated () {
+    this.$nextTick(() => this.updateObserved())
```

Comment:
@techtolentino @korgan00 i think i got it! i was able to resolve the issue using `nextTick` within the `updated` hook. feel free to review, test, check out the [preview link](https://qiskit-org-pr-2427.dcq4xc5i083.us-south.codeengine.appdomain.cloud/) and let me know if you see any problems or have any concerns. thanks.

Reply:
this looks good to me, but I'll wait for @korgan00 

👏🏽 

thanks @vabarbosa

Reply:
Yeah! It looks so good to me. I always forget the `updated` callback.

Amazing! Thank you!